image/mkosi/secure-boot/templates/dev_KEK.conf

```
@@ -0,0 +1,20 @@
+ [ req ]
+ default_bits           = 2048
+ distinguished_name     = req_distinguished_name
+ x509_extensions        = v3_req
+ req_extensions         = v3_req
+ prompt                 = no
+
+ dirstring_type = nobmp
+
+ [ req_distinguished_name ]
+ C                      = DE
+ ST                     = Nordrhein Westfalen
```

```suggestion
 ST                     = Nordrhein-Westfalen
```
But this is a very minor nit and if you have to regenerate the whole PKI just for this, I wouldn't do that.

---

I will leave it as it is since a PKI for testing is already included in github actions.
Interestingly, I found conflicting results on how to format the state field.
[This intermediate CA used by German Telekom](https://de.ssl-tools.net/subjects/fae8baa6c5fe36847b98f212856735704f86280c) uses `Nordrhein Westfalen` while leaf certificates used for https://www.deutschepost.de use `Nordrhein-Westfalen`.